src/pages/instances/InstanceDetailHeaderDropdown.tsx

```diff
@@ -0,0 +1,58 @@
+import { FC } from "react";
+import DeleteInstanceBtn from "./actions/DeleteInstanceBtn";
+import { LxdInstance } from "types/instance";
+import MigrateInstanceBtn from "./actions/MigrateInstanceBtn";
+import { useSettings } from "context/useSettings";
+import { isClusteredServer } from "util/settings";
+import CreateImageFromInstanceBtn from "./actions/CreateImageFromInstanceBtn";
+import DuplicateInstanceBtn from "./actions/DuplicateInstanceBtn";
+import { ContextualMenu } from "@canonical/react-components";
+
+interface Props {
+  instance?: LxdInstance;
+  project: string;
+  isLoading: boolean;
+}
+
+const InstanceDetailHeaderDropdown: FC<Props> = ({
+  instance,
+  project,
+  isLoading,
+}) => {
+  const { data: settings } = useSettings();
+  const isClustered = isClusteredServer(settings);
+
+  return (
+    <>
+      {instance ? (
+        <ContextualMenu toggleLabel="Options" hasToggleIcon visible>
```

Comment:
```suggestion
        <ContextualMenu toggleLabel="Actions" hasToggleIcon visible>
```
I think we should advertise this as actions.

The contextual menu seems to be open when landing on the page. Initially it should be closed.